src/components/action.ts

```diff
@@ -122,14 +184,30 @@ export class Action extends LitElement {
   }
 
   private _setBet(e: MouseEvent): void {
-    this.bet = this._roundToPrecision(
-      (e.target! as BetTarget).multiplier * this.game.pot,
+    let playersPot: number = 0;
+    this.game.players
```

Comment:
Thinking now. We can probably use the `currentBet` on the game Object. and just add the two.

Reply:
Actually, current bet should just hold the previous players bet. nvm